R/data_reverse.R

```diff
@@ -77,12 +83,29 @@ reverse.numeric <- function(x,
     return(x)
   }
 
+  # no missing values allowed
+  if (anyNA(range)) {
+    insight::format_error("`range` is not allowed to have missing values.")
+  }
+
   if (is.null(range)) {
     range <- c(min(x, na.rm = TRUE), max(x, na.rm = TRUE))
   }
 
-  min <- ifelse(is.na(range[1]), min(x, na.rm = TRUE), range[1])
-  max <- ifelse(is.na(range[2]), max(x, na.rm = TRUE), range[2])
+  # check if a valid range (i.e. vector of length 2) is provided
+  if (length(range) > 2) {
+    insight::format_error(
+      "`range` must be a numeric vector of length two, indicating lowest and highest value of the required range.",
+      sprintf(
+        "Did you probably mean to provide `range = c(%g, %g)`?",
+        min(range, na.rm = TRUE),
+        max(range, na.rm = TRUE)
+      )
+    )
+  }
+
+  min <- range[1]
+  max <- range[2]
```

Comment:
should it still use `min()` and `max()` in case someone has `range = c(4, 1)`?

Reply:
It didn't do so before, but we can add this.